tests/scripts/remote_monitoring_tests/json_files/update_results.json

```diff
@@ -3,13 +3,120 @@
     "version": "1.0",
     "experiment_name": "quarkus-resteasy-kruize-min-http-response-time-db",
     "start_timestamp": "2022-01-23T18:25:43.511Z",
-    "end_timestamp": "2022-01-23T18:25:43.511Z",
+    "end_timestamp": "2022-01-23T18:55:43.511Z",
     "kubernetes_objects": [
       {
         "type": "deployment",
         "name": "tfb-qrh-deployment",
         "namespace": "default",
         "containers": [
+	  {
+            "container_image_name": "kruize/tfb-db:1.15",
+            "container_name": "tfb-server-0",
+            "metrics": [
+              {
+                "name": "cpuRequest",
+                "results": {
+                  "value": 1.1,
+                  "format": "cores",
```

Comment:
Suggest that we test the remote monitoring use case with the `resource-usage-openshift` performance profile. It does not have the `value` and `format` for any of the parameters.